tests/conftest.py

```diff
@@ -87,6 +88,26 @@ def pytest_addoption(parser):
         metavar="RUNTIME",
         help="only run tests matching the runtime: [kubernetes].",
     )
+    group.addoption(
+        "--systems",
+        action="store",
+        metavar="SYSTEMS",
+        help="only run tests matching the systems specified [google.cloud, google.marketing_platform]",
+    )
+    group.addoption(
+        "--include-long-lasting",
```

Comment:
```suggestion
        "--include-long-running",
```
Long lasting means something subtly different - since we are talking about the duration of the tests, it is the run (time) that we are describing.

Reply:
Good point. I will update it in my change it comes from #7388

Reply:
Updated in my change.